packages/select/test/visual/lumo/select.test.js

```diff
@@ -119,4 +119,9 @@ describe('select', () => {
     element.setAttribute('theme', 'small');
     await visualDiff(div, 'small');
   });
+
+  it('width', async () => {
+    element.style.width = '80px';
```

Comment:
Might be reasonable to select an item as the displayed value can affect the width as well.

Reply:
Done.